### ISSUE TITLE
[Enhancement] Increase contrast of helper text 

### ISSUE THREAD

**Issue by natalia-fitzgerald** [cfpb/design-system#1702]
Although helper text currently passes AA contrast ratio requirements it is somewhat light, particularly when placed in a form context on top of Gray 5. We can increase the contrast by making the gray darker while will providing the visual distinction that we are seeking for this text. The enhancement I propose here is to update helper text to Dark Gray (#43484e).

## Current behavior
- Inline helper, block helper, and placeholder text
- Avenir Next Regular, 16 px, Gray (#5a5d61)

## Proposed behavior
- Inline helper, block helper, and placeholder text
- Avenir Next Regular, 16 px, Dark Gray (#43484e)

## Screenshots
| Current | Proposed|  
|------------| ------------|
|![Screenshot 2023-08-10 at 1 13 08 PM](https://github.com/cfpb/design-system/assets/77348967/33048350-597e-4ecd-89c3-31b532249447)|![Screenshot 2023-08-10 at 1 12 35 PM](https://github.com/cfpb/design-system/assets/77348967/f9c7b83f-748e-4af7-abe8-c00451a66d92)|

## Changes
- Update color for Inline helper, block helper, and placeholder text to Dark Gray (#43484e)

**Comment by natalia-fitzgerald** [comment]
I'm going to move this one to my team repo as I gather some more examples.

**Comment by chaisamantha** [comment]
I agree with changing the color of the helper text from Gray to Dark Gray. To test the contrast between text color and background, I used the [WebAIM Contrast Checker](https://webaim.org/resources/contrastchecker/).

| Gray on Gray 05 background | Dark Gray on Gray 05 background | 
| ----- | ----- |
| <img width="743" alt="Screenshot 2023-08-11 at 1 33 02 PM" src="https://github.com/cfpb/design-system/assets/109407649/33242599-ace7-442f-90b1-7238eba65375"> | <img width="758" alt="Screenshot 2023-08-11 at 1 32 25 PM" src="https://github.com/cfpb/design-system/assets/109407649/c5cb1fc6-8429-4cb1-b63b-05eb01e06511"> | 

| Gray on Gray 10 background | Dark Gray on Gray 10 background | 
| ----- | ----- |
| <img width="750" alt="Screenshot 2023-08-11 at 1 36 23 PM" src="https://github.com/cfpb/design-system/assets/109407649/d0cd97eb-0e4a-4d92-89bf-7c91d1564808"> | <img width="754" alt="Screenshot 2023-08-11 at 1 37 14 PM" src="https://github.com/cfpb/design-system/assets/109407649/ad2f1110-8450-41a7-8dc2-3e7c75106dd6"> |

Gray does pass the contrast test for WCAG AA, which is the [level that CFPB complies with](https://cfpb.github.io/design-system/guidelines/accessibility-audit-tools). However, it does not pass WCAG AAA. Whenever possible, I think reaching for the optimal accessibility level is best.

**Comment by jenn-franklin** [comment]
@natalia-fitzgerald Great catch and improvement! Thanks for making this change!

**Comment by natalia-fitzgerald** [comment]
## "Before and after" update 
Changes text from Gray (#5a5d61) to Dark Gray (#43484e).

## Pagination
| Before | After |  
|------------| ------------|
|<img width="803" alt="Screenshot 2023-08-14 at 12 20 28 PM" src="https://github.com/cfpb/design-system/assets/77348967/6ee609a4-fadb-4490-b3de-ea01b0f80f8c">|<img width="804" alt="Screenshot 2023-08-14 at 12 20 56 PM" src="https://github.com/cfpb/design-system/assets/77348967/81544e95-e5ab-4eb5-8aec-9f3c89e1c7d2">|

## Inline helper text
| Before | After |  
|------------| ------------|
|<img width="801" alt="Screenshot 2023-08-14 at 12 28 49 PM" src="https://github.com/cfpb/design-system/assets/77348967/662266f4-4778-4766-b04d-7b5dd308f933">|<img width="801" alt="Screenshot 2023-08-14 at 12 29 02 PM" src="https://github.com/cfpb/design-system/assets/77348967/a2430741-58c2-49f6-b8e9-4ac9988a441e">|


### Block helper text
| Before | After |  
|------------| ------------|
|<img width="799" alt="Screenshot 2023-08-14 at 12 28 22 PM" src="https://github.com/cfpb/design-system/assets/77348967/3ed7736b-124e-4d1d-a080-5fb8d8f06917">|<img width="802" alt="Screenshot 2023-08-14 at 12 28 32 PM" src="https://github.com/cfpb/design-system/assets/77348967/f0e3350b-abb4-4b42-8034-6252325dcb19">|

### Placeholder text
| Before | After |  
|------------| ------------|
|<img width="800" alt="Screenshot 2023-08-14 at 12 35 37 PM" src="https://github.com/cfpb/design-system/assets/77348967/bd8e3d01-4d70-4d12-9821-f660a21d2942">|<img width="801" alt="Screenshot 2023-08-14 at 12 36 05 PM" src="https://github.com/cfpb/design-system/assets/77348967/e77da7da-daed-4edb-ba4b-202139bc248f">|


### Disabled button
| Before | After |  
|------------| ------------|
|<img width="808" alt="Screenshot 2023-08-14 at 12 15 12 PM" src="https://github.com/cfpb/design-system/assets/77348967/b5c2be2e-d04d-47d3-b02f-80a52deb1c22">|<img width="806" alt="Screenshot 2023-08-14 at 12 15 04 PM" src="https://github.com/cfpb/design-system/assets/77348967/70b840ca-e52c-4a0a-9d6e-b52fa283ceda">|

### Disabled checkbox
| Before | After |  
|------------| ------------|
|<img width="804" alt="Screenshot 2023-08-14 at 12 16 00 PM" src="https://github.com/cfpb/design-system/assets/77348967/04d25709-9a88-4010-aa99-dfb006991766">|<img width="836" alt="Screenshot 2023-08-14 at 12 15 53 PM" src="https://github.com/cfpb/design-system/assets/77348967/e419b916-ed7f-423c-9e93-6884654f622c">|

### Disabled radio button
| Before | After |  
|------------| ------------|
|<img width="839" alt="Screenshot 2023-08-14 at 12 26 55 PM" src="https://github.com/cfpb/design-system/assets/77348967/e5526b64-4bd2-4152-8966-6c91baa51642">|<img width="837" alt="Screenshot 2023-08-14 at 12 27 08 PM" src="https://github.com/cfpb/design-system/assets/77348967/027ed2ef-8649-4ba9-a93b-73609b8b2c34">|

**Comment by natalia-fitzgerald** [comment]
@meissadia - I am proposing a Design System update that will affect many components that we are building in our component library. My preferred process would be to update the components in the Design System and then pull them in when we pull in an updated version of the Design System. Can our team take on making this change at the source (DS) and then pulling it in with a future release? I can handle the documentation/specs updates in the DS if you can handle the code changes.

**Comment by meissadia** [comment]
@natalia-fitzgerald Sure
